metecho/api/filters.py

```diff
@@ -25,10 +26,17 @@ class Meta:
 
 class TaskFilter(filters.FilterSet):
     slug = filters.CharFilter(field_name="slugs", method=slug_is_active)
+    epic = filters.ModelChoiceFilter(queryset=Epic.objects.all(), null_label="None")
```

Comment:
@jerivas I guess maybe this should be `_("None")`?

Reply:
Makes sense to me 👍